sympy/integrals/__init__.py

```diff
@@ -9,6 +9,7 @@
     >>> integrate(sin(x),x)
     -cos(x)
 """
+from .deltafunctions import deltaintegrate
```

Comment:
Yes. I'm trying to find out why.
Line 98 says

```
.. automethod:: sympy.integrals.deltaintegrate
```

which is the 100% correct name, except that ```deltaintegrate``` is not a method, it's a function.
*However*, line 96 says

```
.. automethod:: sympy.integrals.line_integrate
```

and ```line_integrate`` isn't a method either, so I'm stymied.
(It is not ```sympy.integrals.line_integrate``` but ```sympy.integrals.integrals.line_integrate```, so maybe that ```.rst``` file is simply buggy, but I don't really know what's going on and what would be the right way to fix it.)